src/libsystemd/sd-bus/bus-objects.c

```diff
@@ -1610,6 +1611,102 @@ static int vtable_member_compare_func(const struct vtable_member *x, const struc
 
 DEFINE_PRIVATE_HASH_OPS(vtable_member_hash_ops, struct vtable_member, vtable_member_hash_func, vtable_member_compare_func);
 
+typedef enum {
+        NAMES_FIRSTPART         = 1 << 0, /* first part of argument name list (input names). It is reset by names_are_valid() */
```

Comment:
i think there should be an underscore between FIRST and PART... i.e. `NAMES_FIRST_PART`